tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp

```diff
@@ -77,108 +77,108 @@ void test_frustum_fail() noexcept {
 
 }  // namespace
 
-SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Frustum.Fail", "[Domain][Unit]") {
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Frustum", "[Domain][Unit]") {
   test_frustum_fail();
-}
-
-SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Frustum.Equidistant",
-                  "[Domain][Unit]") {
-  test_suite_for_frustum(false);
-}
+  test_suite_for_frustum(false);  // Equidistant
+  test_suite_for_frustum(true);   // Equiangular
+  SECTION("Alignment") {
```

Comment:
Again, just factor into a separate function and call the function from the main test case